node_modules/col-users/lib/rest.js

```diff
@@ -41,6 +41,26 @@ Collabosphere.apiRouter.get('/users/me', function(req, res) {
 });
 
 /*!
+ * Get user of current course by id
+ */
+Collabosphere.apiRouter.get('/users/:id', function(req, res) {
+  // Get user by id
+  var userId = req.params.id;
+  // Only admins can access non-self user data
+  if (userId != req.ctx.user.id && !req.ctx.user.is_admin) {
+    return res.status(401).send('Unauthorized API request');
+  }
```

Comment:
@paulkerschen Yes, I'm aware that we'll be showing other users a filtered view of the profile. I put this in as a placeholder for "some kind of restriction". I'll remove the placeholder – I'm sure we'll remember said requirement.